.github/workflows/checkstyle.yaml

```diff
@@ -25,7 +25,7 @@ jobs:
       - name: Setup go
         uses: actions/setup-go@v2
         with:
-          go-version: "1.19.3" # Keep in sync with WORKSPACE
+          go-version: "1.20.1" # Keep in sync with WORKSPACE
```

Comment:
There's been a newer version (1.20.3) released since this was created. Might as well update to the latest.

Reply:
Thanks for the catch; done.